app/src/main/java/com/example/workoutapp/MainActivitytwo.kt

```diff
@@ -21,6 +21,7 @@ import androidx.compose.ui.draw.clip
 import androidx.compose.ui.draw.drawBehind
 import androidx.compose.ui.geometry.Offset
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
```

Comment:
Using strings.xml is best practice and gets app ready for internationalization- https://developer.android.com/jetpack/compose/text